docfx/api-override/Meadow.Foundation.mikroBUS/Meadow.Foundation.mikroBUS.Sensors.CLEM.md

```diff
@@ -0,0 +1,25 @@
+---
+uid: Meadow.Foundation.mikroBUS.Sensors.CLEM
+remarks: *content
+---
+
+| CLEM | |
+|--------|--------|
+| Status | <img src="https://img.shields.io/badge/Working-brightgreen" style="width: auto; height: -webkit-fill-available;" alt="Status badge: working" /> |
+| Source code | [GitHub](https://github.com/WildernessLabs/Meadow.Foundation.MikroBus/tree/main/Source/CLEM) |
+| NuGet package | <a href="https://www.nuget.org/packages/Meadow.Foundation.mikroBUS.Sensors.Buttons.CLEM/" target="_blank"><img src="https://img.shields.io/nuget/v/Meadow.Foundation.mikroBUS.Sensors.Buttons.CLEM.svg?label=Meadow.Foundation.mikroBUS.Sensors.Buttons.CLEM" alt="NuGet Gallery for Meadow.Foundation.mikroBUS.Sensors.Buttons.CLEM" /></a> |
+
### Code Example
+
+```csharp
+CLEM currenClick;
+
+public MeadowApp()
+{
+    Console.WriteLine("Initializing ...");
+}
+

```

Comment:
Not remotely critical, but can we trim all the trailing line breaks before we close a code block? (Not just this file, but all, I think.)